### ISSUE TITLE
Clarify project relation or renaming the project?

### ISSUE THREAD

**Issue by hoijui** [codimd/server#204]
Apparently, there are currently two forks of CodiMD on github, with comparable activity:

* this repo [(codimd/server)](https://github.com/codimd/server)
* the original one from HackMD [(hackmdio/codimd)](https://github.com/hackmdio/codimd).

That is pretty unfortunate, as it creates confusion among outsiders, especially (potential) contributors. This channels momentum into the nothing, which is not good for either fork.

I would like to have clarification of the situation and hopefully a discussion about the possible solutions in here.

The related issue in the other repo is ["(hackmdio/codimd) Naming #1219"](https://github.com/hackmdio/codimd/issues/1219).

**Comment by ccoenen** [comment]
We couldn't agree more. But we also don't want to change names _again_.

The short version is: other than the history of the project, there is not really any relationship any more. History has been written up in short in https://github.com/codimd/server/blob/master/docs/history.md

We're not trying to steal their code or their fame. This is why we prominently feature this bit of history in the first few lines in our readme. We're also not trying to make their lifes harder, in fact I wish them well for their business. But I just don't see how this project fits into their strategy.

![Readme Excerpt with the history being mentioned](https://user-images.githubusercontent.com/124909/67158391-bb249b00-f337-11e9-9fd9-cf982918efe8.png)

**Comment by hoijui** [comment]
hmm...
thank you!
I'd still hope you do, but.. I can understand your frustration.

**Comment by edoardoo** [comment]
@ccoenen Sorry to bother, but having two projects which are both very similar and have the same name can only lead to confusion. It looks like both of the projects are missing their own identity. 
Personally, I unintentionally perceived it as a lack of stability. 

I'd be sorry to see a nice open project losing traction because of a naming problem.

**Comment by SISheogorath** [comment]
@edoardoo I definitely agree that it's sad to see this poor thing going on, but I don't think it's on us to change the name. We introduced the name, we developed it to the point that we moved away and this is something the HackMD team didn't want.

To make this clear: We came up with the name, agreed on it, bought the domain, setup an entire brand around it, just to have some people holding the name with the original repo hostile.

I understand that people are unhappy with the situation and I guess there is no one more unhappy about it than I am. But sure, I'm not against collecting yet another round of names for the project.
